inst/rmarkdown/templates/awesomecv/resources/awesome-cv.tex

```diff
@@ -136,9 +136,12 @@
 
 % Print the footer with 3 arguments(<left>, <center>, <right>)
 % Leave any of these blank if they are not needed
+% 2019-02-14 Chris Umphlett - add flexibility to the document name in footer, rather than have it be static Curriculum Vitae
 \makecvfooter
   {$date$}
-  {$name$~~~·~~~Curriculum Vitae}
+  $if(email)$
```

Comment:
`email` should be `footer_docname`
```suggestion
  $if(footer_docname)$
```

Reply:
lol sorry. I did it correctly on my local copy but then goofed when I updated.